.gitignore

```diff
@@ -20,6 +20,8 @@ package-lock.json
 
 # icon list
 src/data/icons.json
+packages/icons/react/index.js
+packages/icons/vue/index.vue
```

Comment:
You need to remove the changes to the icons from your commits.
Your PR does not fix or add any feature to the icon kit right ?
Those files should not be ignored.